Demo/Application/Base/Settings/Settings.bundle/Root.plist

```diff
@@ -19,13 +19,13 @@
 			<array>
 				<string>PayPal - Billing Agreement</string>
 				<string>PayPal - One-Time Payment</string>
```

Comment:
At some point we should also update the use of "Billing Agreement" and "One-Time Payment" in the demo app to be "Vault" and "Checkout". But I'm going to leave that for a separate PR, especially since it isn't necessary for the v5 release.